lib/services/usb-livesync-service.ts

```diff
@@ -118,25 +113,60 @@ export class UsbLiveSyncService extends usbLivesyncServiceBaseLib.UsbLiveSyncSer
 
 			let localProjectRootPath = platform.toLowerCase() === "ios" ? platformData.appDestinationDirectoryPath : null;
 
-			this.sync(platform,
-				this.$projectData.projectId,
-				projectFilesPath,
-				this.excludedProjectDirsAndFiles,
-				watchGlob,
-				platformSpecificLiveSyncServices,
-				notInstalledAppOnDeviceAction,
-				notRunningiOSSimulatorAction,
-				localProjectRootPath,
-				beforeLiveSyncAction,
-				beforeBatchLiveSyncAction,
-				iOSSimulatorRelativeToProjectBasePathAction
-			).wait();
+			let fastLivesyncFileExtensions = [".css", ".xml"];
+			let canExecuteFastLiveSync = (filePath: string) => _.contains(fastLivesyncFileExtensions, path.extname(filePath));
```

Comment:
This could be moved into a dedicated function. The enclosing one is 120 lines long!